workflows/differentialabundance.nf

```diff
@@ -186,13 +186,22 @@ workflow DIFFERENTIALABUNDANCE {
         // We'll be running Proteus once per unique contrast variable to generate plots
         // TODO: there should probably be a separate plotting module in proteus to simplify this
 
-        ch_contrast_variables = ch_contrasts_file
-            .splitCsv(header:true, sep:(params.contrasts.endsWith('csv') ? ',' : '\t'))
-            .map{ it.tail().first() }
-            .map{
-                tuple('id': it.variable)
-            }
-            .unique()   // uniquify to keep each contrast variable only once (in case it exists in multiple lines for blocking etc.)
+        if (params.contrasts.endsWith(".yaml") || params.contrasts.endsWith(".yml")) {
```

Comment:
Is there a need for this conditional if you're fixing things to YML across the board?